### ISSUE TITLE
Keybase: Files need to be rekeyed

### ISSUE THREAD

**Issue by ringmaster** [keybase/client#3286]
I had to reinstall OSX on my Mac, and when I reinstalled keybase, I didn't have access to my paper key.  I then took the obviously foolish step of reissuing the paper key for my account, and now I'm receiving this message whenever I try to access my private kbfs folders:

    Keybase: Files need to be rekeyed
    Please open one of your other computers to unlock /keybase/private/ringmaster

Suffice to say that this Mac was is only system where I had keybase working reliably (`keybase device list` shows that I have one system apart from this Mac with keybase installed, but I haven't figured out how to upgrade that one on Ubuntu past version 0.8.25), and I'd just like it all to work again on this Mac.

My shared private folders seem to work fine, after creating one for a user that I hadn't created a folder for before, but my own private folder insists upon being rekeyed.

I see that others have had their kbfs folders manually reset after doing something dumb like this, and I'll preemptively give you permission to do that, assuming there's no other solution you'd have me try at this time:

> BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5wcIkHbsKDVolM pLwa8d1TE43U9vk lNn2T9cGdoaxtcd siAf3zQuD8XSHMR QccxVJZHcjIS4qc 8JUrHb6RB3RwMA6 znTRd87YUU0afVO OIH9hnoGZRQVX1t kbC63SEyMwoAYmP 0mkFSCMTWYmY2x4 12vkZTDuUSWSnzV SyY6whtT7Zw6yaL 1dsTzsZauWv4foi pzs3IO2VbByG12h 9H0doHfD2DHfxvl 28AS7dBWLTQTVgP 5fnO3sonCHNAwL7 a3OgHdvLct9FvVy 8XVgDgkrWdWdTjp LO3sonCHNAwL9ZK jD6NPO1kwwaBOAl Ap6A8Yxo0fugC57 efMeaqgA8dddj3S 5WzApS5cK36Q9oM BTyT92eYwiNVtBP wR4XhtVvY0m9378 LMfLZoXo8Yo2cOW ojQeIjczFQrWfWQ zNIRrSBNosKBB48 cvvaKDPA2rKuKUB w3NR1tky00. END KEYBASE SALTPACK SIGNED MESSAGE.

Sorry for the trouble, I've already done safe things with my new paper key.

**Comment by maxtaco** [comment]
Thanks for the but report.

A question.  When you reinstalled on OSX, what was the previous version of
keybase?

Even though you reinstalled the software, you should have retained your
previous device key. Did you do anything like manually deleting files?

Thanks.

**Comment by strib** [comment]
Hi @ringmaster -- in theory wiping your OS X install and hence the local keybase files shouldn't revoke your old paper key -- it looks like you purposely did that as part of the new installation?  If your old paper key was still valid, you'd be able to recover the data, but unfortunately it was revoked; you should have been able to make a new paper key as part of the new provisioning without revoking your old paper key.  Do you remember if you explicitly chose to revoke it?

So sadly I think the only option is to reset your folders.  I see your message above, so I can go ahead and do that, but I'll wait for your confirmation again.

**Comment by ringmaster** [comment]
Going through the process of installing Keybase, I didn't realize that I could keep my old paper key and issue a new one.  I feel like I followed default prompts, though I can't be sure I didn't explicitly do something that it was warning me against.  It may be worthwhile reviewing the UI here to ensure that others don't make my same stupid mistake.

Yes, please reset my folders.  Thanks!

**Comment by strib** [comment]
^ @patrickxb @maxtaco: do you know of anything in the provisioning process that would lead someone to revoke their old paper key?

I'll get to the folder resets soon and post back here when it's done.

**Comment by ringmaster** [comment]
Drat.  You know what, this is my fault.  I totally did this at the command line (while trying to figure out what my key was), and hit "y" instead of "n" in response to the "Also revoke existing key" prompt.

I'm an idiot.  Sincerest apologies.

**Comment by patrickxb** [comment]
Right, `keybase paperkey gen` prompts to ask if you'd like to revoke each existing paper key.  

Perhaps we should be clear that it is ok to have multiple before starting that interaction.  The default is no, but still, couldn't hurt to be more clear.

**Comment by strib** [comment]
@ringmaster: ok those folders are reset.  Can you confirm things look ok now?

**Comment by ringmaster** [comment]
> Can you confirm things look ok now?

Yes, works great!  Thanks!

**Comment by strib** [comment]
Cool, sorry for the inconvenience!
